src/Automation/ScanResultsAssembler.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Automation.Interfaces;
```

Comment:
Even though the interface files are in a subdirectory called "interfaces", I believe I've been avoiding actually putting them into a "Interfaces" namespace. So it might be good to follow that pattern.

Reply:
Changed namespace